Source/SharpVectorCore/Utils/PathUtils.cs

```diff
@@ -32,15 +32,15 @@ public static class PathUtils
 		/// <param name="assembly">An <see cref="Assembly"/> which is taken as the base path.</param>
 		/// <returns>A string containing the file name of the assembly.</returns>
 		public static string GetAssemblyFileName(Assembly assembly) =>
-			assembly.ManifestModule.Name;
```

Comment:
Avoid latest C# syntax.

Reply:
- replaced `NET5_0` with `NET50` (already defined in the project file)
- use a *statement body* instead of *lambda* for functions (I hope I understood your comment correctly)